DifferenceGenerator/src/main/kotlin/DifferenceGenerator.kt

```diff
@@ -173,12 +183,43 @@ class DifferenceGenerator(
      * @param color the color
      * @return a Buffered Imnage colored in the given color
      */
-    private fun getColoredBufferedImage(color: Color): BufferedImage {
-        val result = BufferedImage(width, height, BufferedImage.TYPE_3BYTE_BGR)
+    private fun getColoredBufferedImage(
+        color: Color,
+        type: Int = BufferedImage.TYPE_3BYTE_BGR,
+    ): BufferedImage {
+        val result = BufferedImage(width, height, type)
         val g2d: Graphics2D = result.createGraphics()
         g2d.paint = color
         g2d.fillRect(0, 0, width, height)
         g2d.dispose()
         return result
     }
+
+    private fun generateMasking() {
+        if (maskFile == null) {
+            mask = getColoredBufferedImage(Color(255, 255, 255, 0), BufferedImage.TYPE_4BYTE_ABGR)
```

Comment:
Color.WHITE instead?

Reply:
has to be transparent color doesn't matter